spec/features/post_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe "Post" do
 
-  let(:user) { FactoryGirl.create :user }
+  let!(:user) { build :user }
```

Comment:
Hmm, on second thought, I tried running specs having modified it to create instead, and for some reason, it was creating two posts (rather than just one), resulting in the following error thrown by Capybara: "Ambiguous match, found 2 elements matching link or button "-1". I must be missing something, but for now, I've kept it to `build`.